Lombiq.VueJs/Assets/Scripts/helpers/vue-app-compiler-pipeline.js

```diff
@@ -30,32 +36,65 @@ function compile(options) {
     }
 
     return rollupPipeline(
-        opts.destinationPath,
```

Comment:
Can't add this as a suggestion but I would refine the formatting of this function call for increased readability as follows:

```c#
    return rollupPipeline
        (
            opts.appDestinationPath,
            getVueApps(opts.appRootPath).map((appName) => ({
                fileName: appName,
                entryPath: path.join(opts.appRootPath, appName, '/main.js'),
            })),
            [
                json(),
                configureRollupAlias(opts.vueJsNodeModulesPath, opts.isProduction, opts.rollupAlias),
                nodeResolve({ preferBuiltins: true, browser: true, mainFields: ['module', 'jsnext:main'] }),
                replace({
                    values: {
                        'process.env.NODE_ENV': JSON.stringify(opts.isProduction ? 'production' : 'development'),
                        'process.env.BUILD': JSON.stringify('web'),
                    },
                    preventAssignment: true,
                }),
                commonjs(),
            ],
            null,
            (fileName) => path.basename(path.dirname(fileName))
        )
        .catch((error) => {
            if (error) handleErrorObject(error);
            process.exit(1);
        });
```

Reply:
The same in the sfc compiler pipeline.

Reply:
I have never seen the opening parenthesis of a function call moved to the next line. It looks very unsettling and I don't want to do it. Instead I've moved the error handing into the `rollupPipeline`